config/development.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
 module.exports = {
-  PORT: '3000'
+  PORT: '3000',
+  DB_NAME: 'sfmovies_development',
+  DB_HOST: 'localhost',
+  DB_PORT: 5432,
+  DB_PASSWORD: '',
+  DB_USER: 'sfmovies_user'
```

Comment:
It's usually a bit easier to navigate the config files when you sort the keys alphabetically, especially when you have ~50 configs.